Docker/jenkins/Jenkins-CI-Worker/Dockerfile

```diff
@@ -72,7 +72,7 @@ RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
    && chmod a+rx /usr/local/bin/docker-compose
 
 # install nodejs
-RUN curl -sL https://deb.nodesource.com/setup_18.x | bash -
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
```

Comment:
is it ok that the node versions don't match in `Jenkins-CI-Worker`, `Jenkins-Worker` and `Jenkins` images?

Reply:
yes, this is the image used for `gen3.qa.in.a.box`config - for the PR pods (some issue with codeceptjs compatibility). Jenkins normally seems ok with node 18.